packages/edge/docs/README.md

```diff
@@ -0,0 +1,199 @@
+
+Country of the original client IP calculated by Vercel Proxy.
+
+#### Defined in
+
+[src/edge-headers.ts:8](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L8)
+
+---
+
+### IP_HEADER_NAME
+
+• `Const` **IP_HEADER_NAME**: `"x-real-ip"`
+
+Ip from Vercel Proxy. Do not confuse it with the client Ip.
+
+#### Defined in
+
+[src/edge-headers.ts:12](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L12)
+
+---
+
+### LATITUDE_HEADER_NAME
+
+• `Const` **LATITUDE_HEADER_NAME**: `"x-vercel-ip-latitude"`
+
+Latitude of the original client IP calculated by Vercel Proxy.
+
+#### Defined in
+
+[src/edge-headers.ts:16](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L16)
+
+---
+
+### LONGITUDE_HEADER_NAME
+
+• `Const` **LONGITUDE_HEADER_NAME**: `"x-vercel-ip-longitude"`
+
+Longitude of the original client IP calculated by Vercel Proxy.
+
+#### Defined in
+
+[src/edge-headers.ts:20](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L20)
+
+---
+
+### REGION_HEADER_NAME
+
+• `Const` **REGION_HEADER_NAME**: `"x-vercel-ip-country-region"`
+
+Region of the original client IP calculated by Vercel Proxy.
+
+#### Defined in
+
+[src/edge-headers.ts:24](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L24)
+
+## Functions
+
+### geolocation
+
+▸ **geolocation**(`request`): [`Geo`](interfaces/Geo.md)
+
+Returns the location information from for the incoming request
+
+**`See`**
+
+- [CITY_HEADER_NAME](README.md#city_header_name)
+- [COUNTRY_HEADER_NAME](README.md#country_header_name)
+- [REGION_HEADER_NAME](README.md#region_header_name)
+- [LATITUDE_HEADER_NAME](README.md#latitude_header_name)
+- [LONGITUDE_HEADER_NAME](README.md#longitude_header_name)
+
+#### Parameters
+
+| Name      | Type      | Description                                              |
+| :-------- | :-------- | :------------------------------------------------------- |
+| `request` | `Request` | The incoming request object to grab the geolocation data |
+
+#### Returns
+
+[`Geo`](interfaces/Geo.md)
+
+#### Defined in
+
+[src/edge-headers.ts:76](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L76)
+
+---
+
+### ipAddress
+
+▸ **ipAddress**(`request`): `string` \| `undefined`
+
+Returns the IP address of the request from the headers.
+
+**`See`**
+
+[IP_HEADER_NAME](README.md#ip_header_name)
+
+#### Parameters
+
+| Name      | Type      | Description                                     |
+| :-------- | :-------- | :---------------------------------------------- |
+| `request` | `Request` | The incoming request object to grab the IP from |
+
+#### Returns
+
+`string` \| `undefined`
+
+#### Defined in
+
+[src/edge-headers.ts:62](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L62)
+
+---
+
+### next
+
+▸ **next**(`init?`): `Response`
+
+Continue with the request without changing the URL
```

Comment:
Returns a Response that instructs the system to continue processing the request.

Reply:
Could we add an example?

Reply:
done